html/gui/js/CCR.js

```diff
@@ -228,6 +228,133 @@ XDMoD.GlobalToolbar.Contact = function () {
 
 // -------------------------------------------------
 
+XDMoD.createTour = function () {
+    if (typeof XDMoD.tour !== 'undefined') {
```

Comment:
Why not
```js
if (XDMoD.tour !== undefined) {
```
or even
```js
if (XDMoD.tour) {
```

Reply:
Since the previous code (https://github.com/ubccr/xdmod/pull/1021/files#diff-fdf8fdd7e1b5ed7fe7177759254f4351L213) was using this logic I don't see a reason to change it.

```suggestion
    if (XDMoD.tour) {
```

Reply:
im an old programmer, no reason other than that...